addons/stock/migrations/10.0.1.1/post-migration.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trescloud <http://trescloud.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def map_location_auto(cr):
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('auto'), 'auto',
+        [('auto', 'manual')],
+        table='stock_location_path', write='sql')
+
+
+def update_picking_type_id(env):
+    """Updates picking_type_id, defaulting to some value, guessing by looking
+    at source and destination locations, and warehouse.  
```

Comment:
flake8 wants the whitespaces at the end of the line to be removed. there are 9 lines affected.